### DISCUSSION TITLE
Adding image from URL

### DISCUSSION THREAD

**Discussion by premdasvm** [exceljs/exceljs#1976] [Q&A]
Hi, 

I've an column where I need to add image from AWS S3 bucket, how can I do that, I've tried using buffer but it  but it I was showing showing the image on the excel sheet. any way to achieve this?

**Comment by skypesky** [comment]
@premdasvm 
Has your problem been solved?

**Reply by premdasvm** [reply]
No it's hasn't, I came up with a workaround, i download the images to a temp folder then add these images to the excel-sheet while generating, after I'm done, I clear the temp folder. Is there a better way to do this? like directly add images to the excel-sheet via URL?

**Comment by skypesky** [comment]
@premdasvm  see: https://github.com/exceljs/exceljs/issues/1216#issuecomment-1047625753

---
image url -> image buffer -> add to excel file.
